Neos.Flow/Classes/Persistence/Doctrine/CacheAdapter.php

```diff
@@ -84,7 +84,12 @@ public function contains($id)
      */
     public function save($id, $data, $lifeTime = 0)
     {
-        $this->cache->set($this->convertCacheIdentifier($id), $data, [], $lifeTime);
+        try {
+            $this->cache->set($this->convertCacheIdentifier($id), $data, [], $lifeTime);
+            return true;
+        } catch (\Exception $exception) {
+            return false;
+        }
```

Comment:
Mh, I have to say that I share the concerns @albe expressed in #2481 
Silently swallowing exceptions is mostly a bad idea.
Can't we just do

```suggestion
        $this->cache->set($this->convertCacheIdentifier($id), $data, [], $lifeTime);
        return true;
```
?

Reply:
That was my first version of the code I had in mind. But then I realized:

> since this in inside the doctrine adapter, it is only called from the doctrine side, which does expect this to return false on error so... this is probably fine

So in essence I'm okay either way

Reply:
But wouldn't it be helpful to know that a cache entry could not be created because the identifier is invalid for example?
..or if the backend could not be connected to. Those things are almost impossible to debug if they just silently don't work

Reply:
Didn't see this until now, but my vote would have gone to "catch, log, return false". 😎

Reply:
But in which scenario would it be helpful to avoid the exception?

Reply:
Well, if setting the cache entry fails, this is something that might degrade performance, but the result of whatever needed to be cached could very well be valid. Plus, if the calling code expects false in case the operation failed, that should be honoured. Throwing might break error handling further up the stack.

Reply:
Feel free to create a new PR that adds that change on top, I'm not opposed :) For now, the bug is solved and that was my main concern